src/appleseed.studio/python/module.cpp

```diff
@@ -135,4 +136,6 @@ BOOST_PYTHON_MODULE(_appleseedstudio)
 
     bpy::def("main_window", main_window_as_pylong);
     bpy::def("create_dock_widget", create_dock_widget, bpy::args("dock_name"));
+
+    boost::python::def("get_studio_dir", get_executable_directory);
```

Comment:
`get_executable_directory()` should really return the path to the directory containing the executable. It seems sensible to expose it in the Python API. For everything else (in particular for finding resource files) you should use `appleseed::shared::Application::get_root_path()`.